web/locales/en/messages.ftl

```diff
@@ -746,7 +747,6 @@ language = Language
 download-dataset-header = Download the Dataset
 download-delta-explainer = We’ve made some changes. Delta Segments just contain the most recent clips since the last release. <deltaLink>Read more about this work</deltaLink>.
 download-dataset-tag = Select the desired language dataset and choose the version you wish to download.
-download-dataset-access = Looking for previous dataset releases? Email <emailLink> { $email } </emailLink>
```

Comment:
Is it enough to delete the key in the `en` file and change is propagated to the other locales or how does this work?

Reply:
I do believe so, can be confirmed once pontoon is updated